src/Storefront/Resources/views/storefront/component/product/card/box-wishlist.html.twig

```diff
@@ -69,7 +69,7 @@
                 {% endblock %}
 
                 {% block component_product_box_currently_not_available %}
-                    {% set noLongerAvailable = product.isCloseout and product.availableStock < product.minPurchase %}
+                    {% set noLongerAvailable = not product.available %}
 
                     <p class="product-wishlist-info-item product-wishlist-info-not-available">
                         {% if not product.available or noLongerAvailable %}
```

Comment:
This is now redundant :thinking: Might not have been intentional, but I wonder why both checks have been made before when they're the same logically. Please have another look at this and remove the condition in case it actually is equivalent.

Reply:
😆  oh boi. Good catch.

Reply:
Oh, one other thing - since you're checking the logic anyway, it would be great if you could cover this with a small [E2E-test](https://docs.shopware.com/en/shopware-platform-dev-en/references-internals/testing/end-to-end-testing/writing-e2e-tests) :grimacing: The product data, with the availability flag can be added using a [fixture](https://docs.shopware.com/en/shopware-platform-dev-en/references-internals/testing/end-to-end-testing/handling-test-data?category=shopware-platform-dev-en/references-internals/testing/end-to-end-testing). A check of the kind "Is should see `p.product-wishlist-info-not-available` when the product is not available" would suffice here :slightly_smiling_face:

Reply:
Thank you for referencing the docs. I'll have a look. Makes sense to me.